docs/plugins/plugin-events/create-seed.md

```diff
@@ -0,0 +1,51 @@
+---
+id: CreateSeed
+title: Create Seed
+sidebar_label: Create Seed
+slug: /plugins/plugin-events/CreateSeed
+---
+
+# Create Seed
+
+## Description
+
+Creates the seed file. By default, this seed file contains only the seeding of the User entity.
+
+## Event Name:
+`CreateSeed`
+
+## Event Params
+
+```ts
+export interface CreateSeedParams extends EventParams {
+  template: namedTypes.File;
+  templateMapping: { [key: string]: any };
+  fileDir: string;
+  outputFileName: string;
+}
+```
+
+### template
+This is the default template for this event.
+
```

Comment:
Please add here the following line:

This is the default template that is used for this event.
https://github.com/amplication/amplication/blob/next/packages/amplication-data-service-generator/src/server/seed/seed.template.ts

Reply:
corrected